docs/MAINTAINERS.md

```diff
@@ -1,8 +0,0 @@
-## Maintainers
```

Comment:
This file doesn't tell the truth.

Kubernetes holds a OWNERS file strategy for similar usage and etcd maintain the MAINTAINERS files with a few maintainers in fact. We have a large number of maintainer and possibly build a OWNERS like org to maintain it.

Reply:
revert change, defer until we actually implement OWNERS file org.